src/pill/types.ts

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+export type Type = "outline" | "solid";
```

Comment:
Rename to `PillType` since this is exposed all the way to root level. We want to eliminate name clash or confusion